airbyte-integrations/connectors/source-salesforce/source_salesforce/spec.json

```diff
@@ -4,41 +4,41 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Salesforce Source Spec",
     "type": "object",
-    "required": [
-      "client_id",
-      "client_secret",
-      "refresh_token",
-      "start_date",
-      "api_type"
-    ],
+    "required": ["client_id", "client_secret", "refresh_token", "api_type"],
     "additionalProperties": false,
     "properties": {
       "client_id": {
+        "title": "Client Id",
         "description": "The Consumer Key that can be found when viewing your app in Salesforce",
         "type": "string"
       },
       "client_secret": {
+        "title": "Client Secret",
         "description": "The Consumer Secret that can be found when viewing your app in Salesforce",
         "type": "string",
         "airbyte_secret": true
       },
       "refresh_token": {
+        "title": "Refresh Token",
         "description": "Salesforce Refresh Token used for Airbyte to access your Salesforce account. If you don't know what this is, follow this <a href=\"https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b\">guide</a> to retrieve it.",
         "type": "string",
         "airbyte_secret": true
       },
       "start_date": {
-        "description": "UTC date and time in the format 2017-01-25T00:00:00Z. Any data before this date will not be replicated. This field uses the \"updated\" field if available, otherwise the \"created\" fields if they are available for a stream.",
+        "title": "Start Date",
+        "description": "Date in the format 2017-01-25. Any data before this date will not be replicated. This field uses the \"updated\" field if available, otherwise the \"created\" fields if they are available for a stream.",
         "type": "string",
-        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
-        "examples": ["2021-07-25T00:00:00Z"]
+        "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z|[0-9]{4}-[0-9]{2}-[0-9]{2}$",
+        "examples": ["2021-07-25", "2021-07-25T00:00:00Z"]
       },
       "is_sandbox": {
+        "title": "Is Sandbox?",
         "description": "Whether or not the the app is in a Salesforce sandbox. If you do not know what this, assume it is false. We provide more info on this field in the <a href=\"https://docs.airbyte.io/integrations/destinations/salesforce#is_sandbox\">docs</a>.",
         "type": "boolean",
         "default": false
       },
       "api_type": {
```

Comment:
can you create an issue to deprecate this option? this is an implementation detail that we should remove eventually

Reply:
It's created [here](https://github.com/airbytehq/airbyte/issues/8419)